messaging/src/main/java/org/axonframework/serialization/upcasting/event/IntermediateEventRepresentation.java

```diff
@@ -139,4 +140,18 @@
      * @return the MetaData of the message wrapping the object to upcast, if available
      */
     LazyDeserializingObject<MetaData> getMetaData();
+
+    /**
+     * Checks if the data can be converted to required type.
+     *
+     * @param requiredType the type to convert to
+     * @return true, if the intermediate representation can be converted to desired type.
+     */
+    <D> Boolean canConvertDataTo(Class<D> requiredType);
```

Comment:
see above. Please use `boolean` instead.

Reply:
fixed in 64b00fd50f3aac8de0261bfc718c44e919abcd43